[INT-201] Visual snapshots CLI - templates and defaults (#3158)
saucelabs/sauce-docs: docs/visual-testing/integrations/snapshots-cli.md

- `--suite-name`: The name of the suite you would like to appear in the Sauce Visual dashboard. Default: `undefined` (appears as `Unknown` in the Sauce Visual dashboard). Supports the following parameters: `{filename}, {ext}, {directory}, {directoryRelative}, {page}`. Default: `{directoryRelative}`.
- `--test-name`: The name of the test you would like to appear in the Sauce Visual dashboard. Supports the following parameters: `{filename}, {ext}, {directory}, {directoryRelative}, {page}`. Default: `{filename}{ext}`.
- `--snapshot-name` The name of the snapshot you would like to appear in the Sauce Visual dashboard. Supports the following parameters: `{filename}, {ext}, {directory}, {directoryRelative}, {page}`. Default: `{filename}-page-{page}`.
- `-j`, `--concurrency`: Maximum count of simultaneous uploads. Default: the number of CPUs.

##### Suite name, test name and snapshots name paameters

- `filename` - name of the PDF file without the extension
- `ext` - file extension (_.pdf_)

> - `page` - page number, staring from 1

Reviewer: ```suggestion
- `page` - page number, starting from 1
```

> - `directory` - name of the directory the PDF file is located at

Reviewer: ```suggestion
- `directory` - name of the directory containing the PDF file
```

> - `directoryRelative` - relative path from the current directory to the file (excluding the filename) 

Reviewer: ```suggestion
- `directoryRelative` - relative path from current working directory to the directory containing the PDF file
```
